docker-compose.yml

```diff
@@ -88,7 +88,7 @@ services:
       - kafka
 
   db:
-    image: quay.io/debezium/postgres:12
+    image: quay.io/debezium/postgres:14
     restart: always
```

Comment:
btw, in `PROD`, I think we are using `Postgres:12`

Reply:
I'll revert back to '12' and let the tests run with it

Reply:
tried locally and still works fine.